simpleclient_hotspot/src/main/java/io/prometheus/client/hotspot/StandardExports.java

```diff
@@ -56,8 +58,19 @@ public StandardExports() {
   public List<MetricFamilySamples> collect() {
     List<MetricFamilySamples> mfs = new ArrayList<MetricFamilySamples>();
 
-    mfs.add(new CounterMetricFamily("process_cpu_seconds_total", "Total user and system CPU time spent in seconds.",
-        osBean.getProcessCpuTime() / NANOSECONDS_PER_SECOND));
+    try {
+      Method method = osBean.getClass().getMethod("getProcessCpuTime", null);
+      if (!method.isAccessible()) {
```

Comment:
The method, yes, not the class. So the if test will fail, it will go inside, and set it accessible. Without it it will fail...